pkg/operation/botanist/component/shared/kubeapiserver.go

```diff
@@ -125,8 +125,8 @@ func NewKubeAPIServer(
 	)
 
 	if apiServerConfig != nil {
-		enabledAdmissionPlugins = computeEnabledKubeAPIServerAdmissionPlugins(enabledAdmissionPlugins, apiServerConfig.AdmissionPlugins)
-		disabledAdmissionPlugins = computeDisabledKubeAPIServerAdmissionPlugins(apiServerConfig.AdmissionPlugins)
+		enabledAdmissionPlugins = computeEnabledKubeAPIServerAdmissionPlugins(enabledAdmissionPlugins, apiServerConfig.AdmissionPlugins, isNodeless)
```

Comment:
Sometimes we call it "nodeless", sometimes we call it "workerless". Please harmonize this across the entire code base.

Reply:
Not sure, why would we introduce such rule? This will easily go "out of sync" over time, I believe. Nodes and workers are similarly used in the Kubernetes environment, so either of it is fine, but I still think we should decide for only one (unless there is a good reason to distinguish it as proposed above).

Reply:
Then let's go with workerless.